src/core/mempool.rs

```diff
@@ -255,6 +281,28 @@ const MEMPOOL_INITIAL_SCHEMA: &'static [&'static str] = &[
     "CREATE INDEX by_chaintip ON mempool(consensus_hash,block_header_hash);",
 ];
 
+const MEMPOOL_SCHEMA_2_TEST: &'static str = "
+   SELECT name FROM sqlite_master WHERE type='table' AND name='fee_estimates'
+";
+
+const MEMPOOL_SCHEMA_2: &'static [&'static str] = &[
+    r#"
+    CREATE TABLE fee_estimates(
+        txid TEXT NOT NULL,
+        fee_rate NUMBER,
+        PRIMARY KEY (txid),
+        FOREIGN KEY (txid) REFERENCES mempool (txid) ON DELETE CASCADE ON UPDATE CASCADE
+    );
+    "#,
+    r#"
+    ALTER TABLE mempool ADD COLUMN last_known_origin_nonce INTEGER;
+    "#,
+    r#"
+    ALTER TABLE mempool ADD COLUMN last_known_sponsor_nonce INTEGER;
+    "#,
+    "CREATE INDEX fee_by_txid ON fee_estimates(txid);",
+];
```

Comment:
Can you also update the mempool DB schema so that these columns simply get created in the table if the table is being made for the first time?

Reply:
Also, is the mempool DB versioned?  If not, we should add it, so we know what migrations to apply.